File: android/app/src/main/java/com/tonhub/wallet/modules/store/PostEncryptionCallback.java
package com.tonhub.wallet.modules.store;

import com.facebook.react.bridge.Promise;

import org.json.JSONException;

import java.security.GeneralSecurityException;

public interface PostEncryptionCallback {
    void run(Promise promise, Object result) throws JSONException, GeneralSecurityException;
}

File: android/app/src/main/java/com/tonhub/wallet/modules/store/PostEncryptionCallback.kt
package com.tonhub.wallet.modules.store

import com.facebook.react.bridge.Promise
import org.json.JSONException
import java.security.GeneralSecurityException

interface PostEncryptionCallback {
    @Throws(JSONException::class, GeneralSecurityException::class)
    fun run(promise: Promise, result: Any?)
}


